### DISCUSSION TITLE
Add second shift only if the employee worked the 1st shift in conjunction with weekly cover demands

### DISCUSSION THREAD

**Discussion by PredatorXix** [google/or-tools#3597] [CP-SAT questions]
Currently an employee can work a max of 3 consecutive days- working

An employee can work between 45 and 60 hours a week- working

And I am getting close to my goal I am running into issue regarding a constraint that an employee can only work the extended shift if they worked the day shift.

**Comment by lperron** [comment]
So not use NotBoolVar, always use Literal.

**Reply by lperron** [reply]
Do not...

**Reply by PredatorXix** [reply]
Thank you
